docs/assets/js/inject-editor.js

```diff
@@ -55,7 +58,7 @@ function getTheme() {
     { token: "string", foreground: text4 },
     { token: "keyword", foreground: text6 }
   ];
-  const base = getUserColorTheme() === "light" ? "vs" : "vs-dark";
+  const base = userColorTheme === "light" ? "vs" : "vs-dark";
```

Comment:
I don't know what the implications of `base` in the return object is, but when you change the theme, this property is not updated, i.e.
- Load the page in light mode, `base` is `vs`
- Click the theme, changes to dark, `base` is `vs`

I didn't see any obvious impact, but did confirm that the response from `getTheme()` is incorrect in `setupColorThemeListener()`

Reply:
Ah, I see now that's a global you are setting at startup time, so maybe it's intentional?

Reply:
It wasn't, but it looks like we don't actually need the fallback theme to be correct, in practice, at least not in an obvious way. Regardless, I've updated it to change when the theme changes.